### DISCUSSION TITLE
Using fields query parameter on showing a single resource

### DISCUSSION THREAD

**Discussion by aspyropoulos** [spatie/laravel-query-builder#804] [Q&A] [answered]
I want to query a single resource out of which I only need one field (title).
Is it a good practice to use the fields query parameter on showing a single resource? Is this a violation of the JSON API specification?
ie `customers/1?fields[customers]=title`

**Comment by dominikb** [comment, marked as answer]
As far as I can tell its exactly what you want a `fields` parameter to do: https://jsonapi.org/format/#fetching-sparse-fieldsets
